egs2/README.md

```diff
@@ -12,6 +12,7 @@ See: https://espnet.github.io/espnet/espnet2_tutorial.html#recipes-using-espnet2
 | aishell                 | AISHELL-ASR0009-OS1 Open Source Mandarin Speech Corpus                                                                           | ASR                     | CMN                   | http://www.aishelltech.com/kysjcp                                                                            |              |
 | aishell3                | AISHELL3 Mandarin multi-speaker text-to-speech                                                                                   | TTS                     | CMN                   | https://www.openslr.org/93/                                                                                  |              |
 | aishell4                | AISHELL4 Open Source Mandarin Speech Corpus in Conference Scenario                                                               | ASR/SE                  | CMN                   | https://www.openslr.org/111/                                                                                 |              |
+| americasnlp22           | The Second AmericasNLP Competition                                                                                               | ASR                     | 5 languages       | http://turing.iimas.unam.mx/americasnlp/st.html                                                                  |              |
```

Comment:
Can you add a three-letter language code for these languages?

Reply:
Done!